azurerm/resource_arm_stream_analytics_output_eventhub_test.go

```diff
@@ -97,6 +97,35 @@ func TestAccAzureRMStreamAnalyticsOutputEventHub_json(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMStreamAnalyticsOutputEventHub_jsonArrayFormat(t *testing.T) {
+	resourceName := "azurerm_stream_analytics_output_eventhub.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMStreamAnalyticsOutputEventHubDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMStreamAnalyticsOutputEventHub_jsonArrayFormat(ri, location),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMStreamAnalyticsOutputEventHubExists(resourceName),
+				),
```

Comment:
We should probably add some checks here:
```go
resource.TestCheckResourceAttr(resourceName, "serialization.0.format", "Array"),
```

and for the rest of the serialization block

Reply:
done here and for all other types of serialization